docs/conf.py

```diff
@@ -0,0 +1,275 @@
+# Usually you set "language" from the command line for these cases.
+language = 'en'
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+html_theme = 'furo'
+
+html_show_sphinx = True
+
+html_theme_options = {
+}
+
+html_context = {
+}
+
+
+# The name for this set of Sphinx documents.  If None, it defaults to
+# "<project> v<release> documentation".
+# html_title = f'{project} Documentation'
+
+# A shorter title for the navigation bar.  Default is the same as html_title.
+# html_short_title = 'Documentation'
+
+# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
+# using the given strftime format.
+html_last_updated_fmt = '%b %d, %Y'
+
+# If true, an OpenSearch description file will be output, and all pages will
+# contain a <link> tag referring to it.  The value of this option must be the
+# base URL from which the finished HTML is served.
+# html_use_opensearch = f'https://{github_repo_name.replace(".", "")}.readthedocs.io/en/latest/'
+
+# The master toctree document.
+root_doc = master_doc = 'index'  # Sphinx 4+ / 3-  # noqa: WPS429
+
+
+# -- Extension configuration -------------------------------------------------
+
+# -- Options for intersphinx extension ---------------------------------------
+
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+    'python2': ('https://docs.python.org/2', None),
+}
+
+# -- Options for todo extension ----------------------------------------------
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True
+
+# -- Options for sphinxcontrib.apidoc extension ------------------------------
+
+apidoc_excluded_paths = [
+    'plugin/cache/*',
+    'testing/*.py',
+]
+apidoc_extra_args = [
+    '--implicit-namespaces',
+    '--private',  # include “_private” modules
+]
+apidoc_module_dir = str(PROJECT_ROOT_DIR / 'proxy')
+apidoc_module_first = False
+apidoc_output_dir = 'pkg'
+apidoc_separate_modules = True
+apidoc_toc_file = None
+
+# -- Options for sphinxcontrib.spelling extension ----------------------------
+
+spelling_ignore_acronyms = True
+spelling_ignore_importable_modules = True
+spelling_ignore_pypi_package_names = True
+spelling_ignore_python_builtins = True
+spelling_ignore_wiki_words = True
+spelling_show_suggestions = True
+spelling_word_list_filename = [
+    'spelling_wordlist.txt',
+]
+
+# -- Options for extlinks extension ------------------------------------------
+
+extlinks = {
+    'issue': (f'{github_repo_url}/issues/%s', '#'),  # noqa: WPS323
+    'pr': (f'{github_repo_url}/pull/%s', 'PR #'),  # noqa: WPS323
+    'commit': (f'{github_repo_url}/commit/%s', ''),  # noqa: WPS323
+    'gh': (f'{github_url}/%s', 'GitHub: '),  # noqa: WPS323
+    'user': (f'{github_sponsors_url}/%s', '@'),  # noqa: WPS323
```

Comment:
@abhinavsingh these are simple roles for making external references. For example, you can use `user` (in RST) in as follows:
```rst
:user:`webknjaz`
```
And that will link to my sponsors page (for users w/o sponsors enabled, GH will automatically redirect to their profiles).
This is useful to credit contributions to people in the changelog, for example.